PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/mysql/creation/MySQLCreationDialog.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.azure.toolkit.intellij.mysql.creation;
+
+import com.intellij.openapi.project.Project;
+import com.microsoft.azure.management.resources.ResourceGroup;
+import com.microsoft.azure.management.resources.Subscription;
+import com.microsoft.azure.toolkit.intellij.common.AzureDialog;
+import com.microsoft.azure.toolkit.lib.common.form.AzureForm;
+import com.microsoft.azure.toolkit.lib.mysql.AzureMySQLConfig;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ItemEvent;
+import java.awt.event.ItemListener;
+
+import static com.microsoft.intellij.ui.messages.AzureBundle.message;
+
+public class MySQLCreationDialog extends AzureDialog<AzureMySQLConfig> {
+    private static final String DIALOG_TITLE = "Create Azure Database for MySQL";
+    private JPanel rootPanel;
+    private MySQLCreationBasic basic;
+    private MySQLCreationAdvanced advanced;
+
+    private boolean advancedMode;
+    private JCheckBox checkboxMode;
+
+    public MySQLCreationDialog(@Nullable Project project) {
+        super(project);
+        setOKActionEnabled(false);
+        init();
+        extendInit();
+        initListeners();
+    }
+
+    private void extendInit() {
+        advanced.setVisible(false);
+        this.startTrackingValidation();
+    }
+
+    private void initListeners() {
+        advanced.getSubscriptionComboBox().addItemListener(new ItemListener() {
```

Comment:
could we skip these listener as we will getData from advancedPanel and set it to basicPanel while switching?